JavaAssignments/src/saket/Studentdetails.java

```diff
@@ -0,0 +1,17 @@
+package saket;
+
+public class Studentdetails {
+
+	public static void main(String[] args) {
+
+		System.out.println("Roollnum is " + args[0]);
+
+	    System.out.println("Name of the Student is " + args[1]);
+
+		int i = Integer.parseInt(args[2]);
+		System.out.println("marks of the student " + args[1] + " is " + i);
```

Comment:
You should write two methods.
1) setData which will set Student data.
2) display method which will print it.